### PR TITLE
🌱 Update CR v0.9.0-alpha.0 and golint 1.32 dependencies

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

/assign @CecileRobertMichon @fabriziopandini

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -19,7 +19,6 @@ linters:
     - gosimple
     - govet
     - ineffassign
-    - maligned
```
Comment (vincepri) [inline .golangci.yml:22]: This linter has been deprecated

**Review by fabriziopandini — COMMENTED** [review]
/lgtm

**Pull request merged**